src/Connections/FeedConnection.ts

```diff
@@ -182,6 +185,15 @@ export class FeedConnection extends BaseConnection implements IConnection {
     }
 
     public async handleFeedEntry(entry: FeedEntry): Promise<void> {
+        // We will need to tidy this up.
+        if (this.state.template?.includes('$SUMMARY') && entry.summary) {
+            // This might be massive and cause us to fail to send the message
+            // so confine to a maximum size.
+            if (entry.summary.length > MAX_SUMMARY_LENGTH) {
+                entry.summary = entry.summary?.substring(0, MAX_SUMMARY_LENGTH) + "…" ?? null;
+            }
+            entry.summary = sanitizeHtml(entry.summary);
```

Comment:
Does `sanitizeHtml` (or rather, ruma's `sanitize_html`) fail when given HTML with unclosed tags?  That can happen since `entry.summary` may be trimmed.

Reply:
Added a test, looks happy with unclosed tags :+1: